internal/prompt/prompt.go

```diff
@@ -318,18 +274,31 @@ func Seed(reader *bufio.Reader) (seed []byte, imported bool, err error) {
 				fmt.Printf("Input error: %v\n", err.Error())
 			}
 		} else {
-			seed, err = walletseed.DecodeUserInput(seedStrTrimmed)
+
+			useMnemonicPassphrase, err := promptListBool(reader, "Do you use a "+
+				"BIP39 passphrase?", "no")
+			if err != nil {
+				return nil, true, err
+			}
+
+			var mnemonicPassphrase string
+			if useMnemonicPassphrase {
```

Comment:
This could be refactored into a function so `mnemonicPassphrase` is created and initalized in one line?

Reply:
Yes, that's a good idea :+1: